osgp/platform/osgp-adapter-domain-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/domain/da/application/services/DeviceManagementService.java

```diff
@@ -86,10 +90,25 @@ public void handleHealthStatusResponse(final GetHealthStatusResponseDto getHealt
             exception = this.ensureOsgpException(e, "Exception occurred while getting Health Status Response Data");
         }
 
-        ResponseMessage responseMessage = ResponseMessage.newResponseMessageBuilder()
-                .withCorrelationUid(correlationUid).withOrganisationIdentification(organisationIdentification)
-                .withDeviceIdentification(deviceIdentification).withResult(result).withOsgpException(exception)
-                .withDataObject(getHealthStatusResponse).build();
+        final ResponseMessage responseMessage = ResponseMessage.newResponseMessageBuilder()
+                .withCorrelationUid(correlationUid)
+                .withOrganisationIdentification(organisationIdentification)
+                .withDeviceIdentification(deviceIdentification)
+                .withResult(result)
+                .withOsgpException(exception)
+                .withDataObject(getHealthStatusResponse)
+                .build();
         this.webServiceResponseMessageSender.send(responseMessage, messageType);
     }
+
+    public void addMeter(final DeviceMessageMetadata deviceMessageMetadata,
+            final AddRtuDeviceRequest addRtuDeviceRequest) throws FunctionalException {
+        final String organisationIdentification = deviceMessageMetadata.getOrganisationIdentification();
+        final String deviceIdentification = deviceMessageMetadata.getDeviceIdentification();
+        LOGGER.debug("addMeter for organisationIdentification: {} for deviceIdentification: {}", organisationIdentification,
+                deviceIdentification);
+        this.rtuDeviceService.storeRtuDevice(organisationIdentification, addRtuDeviceRequest);
+        final RtuDevice rtuDevice = addRtuDeviceRequest.getRtuDevice();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "rtuDevice" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1481)

Reply:
Done